src/System.Net.Http/src/System/Net/Http/HPack/HPackDecodingException.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace System.Net.Http.HPack
+{
+    // TODO: Should this be public?
```

Comment:
Do we ever end up throwing this out where it could be caught, or will it always end up getting wrapped by something?

In general, as you I and spoke about earlier this week, we need to revamp our whole error reporting story for System.Net.Http.  This could potentially be part of that.

Reply:
Currently, it's not wrapped. It probably should be.